src/spritekit.cs

```diff
@@ -2372,5 +2558,372 @@ interface SKReferenceNode {
 		[Export ("resolveReferenceNode")]
 		void Resolve ();
 	}
+
+	[iOS (9,0)][Mac (10,11)]
```

Comment:
where the introspection tests executed on iOS 9.0 ?
we should see how the bots, running El Capitan, will react for the Mac side